### DISCUSSION TITLE
New Team Member: Welcome

### DISCUSSION THREAD

**Discussion by kebekus** [Akaflieg-Freiburg/enroute#461] [Announcements]
@Akaflieg-Freiburg/enroute 

Dear all,

I want to introduce Tom Linz alias @Munzijoy to our team, who has kindly provided code to compute density heights from METAR data. Welcome!

Best wishes,

Stefan.

**Comment by TTC70** [comment]
Cheers and, @Munzijoy  welcome to the team!!
Good good, the family is getting bigger!!
